src/app/pages/apps/components/chart-wizard/chart-wizard.component.ts

```diff
@@ -161,41 +160,43 @@ export class ChartWizardComponent implements OnInit, AfterViewInit, OnDestroy {
         this.cdr.markForCheck();
 
         if (this.activatedRoute.routeConfig.path.includes('install')) {
-          this.makeChartCreate();
+          this.loadApplicationForCreation();
```

Comment:
I think checking the route path for keywords could be flaky. Since the route contains app name, an app name can potentially have "install" or "edit" and break the UI. A better approach would be to have a shared component "form" and separate components for install and edit that use the "form" component and set it up to their need with input properties. Then set the routes components correctly based on what we want.

Reply:
@RehanY147 
I think we can update that `includes` part with better/precise check
like here:
<img width="578" alt="Screenshot 2023-06-13 at 12 39 53" src="https://github.com/truenas/webui/assets/22980553/8ff20d41-cdb4-477d-aa40-b34b16fb9d5c">

Reply:
In this case it will always work since the route can ends only with these strings.